src/module/data/actor-character.mjs

```diff
@@ -45,12 +56,37 @@ export default class GrimwildCharacter extends GrimwildActorBase {
 						max: 4,
 						initial: 1,
 						min: 0
-					})
+					}),
+					marked: new fields.BooleanField()
 				});
 				return obj;
 			}, {})
 		);
 
+		schema.thorns = new fields.NumberField({
+			integer: true,
+			min: 0
+		});
+
+		schema.features = new fields.StringField();
+		schema.conditions = new fields.StringField();
+		schema.backgrounds = new fields.SchemaField({
+			name: new fields.StringField(),
+			wises: new fields.ArrayField(new fields.StringField())
+		});
+		schema.traits = new fields.SchemaField({
```

Comment:
I would lean towards flipping this so that it's an array of schema fields like:

```js
{
  traits: [
    { are: true, value: 'Brave' },
    { are: true, value: 'Persistent' },
    { are: false, value: 'Quiet' }
  ]
}
```

We're going to have a lot of things that use the pick two that are true and one that isn't setup, so having the consistent structure will make it easier to drill down them for query and template purposes.